From 13d208c4ce0cb31b3037a16cf35591c894971c6a Mon Sep 17 00:00:00 2001
From: Danny Kukawka <danny.kukawka@web.de>
Date: Thu, 30 Jul 2009 17:43:29 +0200
Subject: [PATCH 05/19] add another suspend quirks for IBM machines

Added another suspend quirks for IBM machines. Tested with
kernel 2.6.27.19.

from: bnc#488035
---
 .../10freedesktop/20-video-quirk-pm-ibm.fdi        |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/fdi/information/10freedesktop/20-video-quirk-pm-ibm.fdi b/fdi/information/10freedesktop/20-video-quirk-pm-ibm.fdi
index 4d4a2b4..011d763 100644
--- a/fdi/information/10freedesktop/20-video-quirk-pm-ibm.fdi
+++ b/fdi/information/10freedesktop/20-video-quirk-pm-ibm.fdi
@@ -122,7 +122,7 @@
     </match>
 
     <match key="system.hardware.vendor" string="IBM CORPORATION">
-      <match key="system.hardware.product" string_outof="4800723;4800743;4800783">
+      <match key="system.hardware.product" string_outof="4800723;4800743;4800783;4800C43">
         <merge key="power_management.quirk.s3_bios" type="bool">true</merge>
         <merge key="power_management.quirk.s3_mode" type="bool">true</merge>
       </match>
-- 
1.6.4.4

